src/dbridge_js/ditecx/sign-request.ts

```diff
@@ -94,6 +94,7 @@ export class SignRequest {
       transformationTargetEnvironment: getNullIfEmpty(
         this.objectInfo.transformationTargetEnvironment
       ),
+      embedUsedSchemas: !this.objectInfo.includeRefs,
```

Comment:
Ukázalo sa, že takto to vlastne sedí. "Include references" znamená, že sa v XDC vyrobia iba referencie na XSD a XSLT. My toto máme default a práveže posielame Autogramu, či ich mám embedovať (teda opak referencovania).